web/profiles/custom/yalesites_profile/modules/custom/ys_templated_content/ys_templated_content.routing.yml

```diff
@@ -0,0 +1,7 @@
+ys_templated_content.selection:
+  path: '/admin/node/add/from_template'
+  defaults:
+    _form: '\Drupal\ys_templated_content\Form\TemplatedContentForm'
+    _title: 'Create content from a template'
+  requirements:
+    _permission: 'create page content'
```

Comment:
This works, but may not be true in the future. I can foresee a time where we have additional roles and permissions to create different types of contributors. For example, someone who can contribute to the blog but is not allowed to create new landing pages. With this in mind, I would either:

Pick a more generic permission, such as 'access content overview'
OR create a new permission for 'create content from template' and give this permission to the appropriate roles.

Reply:
Nice; ya I'll work on just creating a specific permission for this so we can be explicit about that functionality.  Thanks a lot for looking.  I'll work on these updates this week so we can be ready.